Development/nmos/control_protocol_handlers.h

```diff
@@ -38,23 +38,15 @@ namespace nmos
 
     namespace experimental
     {
-        // standard method handler definition
-        typedef std::function<web::json::value(nmos::resources& resources, const nmos::resource& resource, int32_t handle, const web::json::value& arguments, bool is_deprecated, get_control_protocol_class_descriptor_handler get_control_protocol_class_descriptor, get_control_protocol_datatype_descriptor_handler get_control_protocol_datatype_descriptor, control_protocol_property_changed_handler property_changed, slog::base_gate& gate)> standard_method_handler;
-
-        // non-standard method handler definition
-        typedef std::function<web::json::value(nmos::resources& resources, const nmos::resource& resource, int32_t handle, const web::json::value& arguments, bool is_deprecated, slog::base_gate& gate)> non_standard_method_handler;
+        // control method handler definition
+        typedef std::function<web::json::value(nmos::resources& resources, const nmos::resource& resource, const web::json::value& arguments, bool is_deprecated, slog::base_gate& gate)> method_handler;
```

Comment:
`method_handler` feels a little generic to me.  Could we call this `control_protocol_method_handler` instead?

Reply:
sure, let me change it